### PR TITLE
imagebuildah: add AdditionalTags

### PR DESCRIPTION
**Pull request by nalind**
Add an AdditionalTags field to the imagebuildah options structure, to provide for additional tags which we'll want to apply to the image that we're writing.

### CODE REVIEW

File: commit.go [hunk 1]
```diff
@@ -37,5 +67,20 @@ func (b *Builder) Commit(dest types.ImageReference, options CommitOptions) error
 		return err
 	}
 	err = copy.Image(policyContext, dest, src, getCopyOptions())
+	switch dest.Transport().Name() {
+	case storage.Transport.Name():
+		tags, err := expandTags(options.AdditionalTags)
+		if err != nil {
```
Comment (TomSweeneyRedHat) [inline commit.go]: Do you want to also spin off some kind of error message for these similar to what you did above in expandTags?  Or will the returned err have enough info in it to tell which of these three conditions the error sprang from?

Comment (nalind) [inline commit.go]: Yeah, it should be doing that for the SetNames() call, at least.  I'll fix that.

**Comment by nalind** [comment]
Added some context to the error message returned if we get an error from SetNames().

**Comment by nalind** [comment]
Removed a redundant type assertion that @TomSweeneyRedHat spotted out of band.

File: commit.go (hunk 1 above)
Comment (rhatdan) [inline commit.go]: Did you fix this?

Comment (nalind) [inline commit.go]: Yes, not long after.

**Comment by rhatdan** [comment]
@rh-atomic-bot r+ 3c3174c

**Pull request closed without merging**